src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExtensionMethodImportCompletionProviderTests.cs

```diff
@@ -25,6 +25,8 @@ public ExtensionMethodImportCompletionProviderTests(CSharpTestWorkspaceFixture w
         {
         }
 
+        private const string NonBreakingSpaceString = "\x00A0";
```

Comment:
?

Reply:
This is the char used in the product, took me some debugging to figure out why my tests were failing.